core_backend/migrations/versions/2024_08_01_465368ca2bac_rename_tables_and_add_user_id_column.py

```diff
@@ -0,0 +1,223 @@
+"""Rename tables and add user_id column
+
+Revision ID: 465368ca2bac
+Revises: 9d0519332fdd
+Create Date: 2024-08-01 20:36:52.629572
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "465368ca2bac"  # pragma: allowlist secret
+down_revision: Union[str, None] = "9d0519332fdd"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # Rename tables
```

Comment:
The DDL test was failing. I looked at the DDL statement of upgraded tables and added the index operations.